demo/Droid/MvvmNanoDemo.Droid.csproj

```diff
@@ -38,6 +38,14 @@
     <ConsolePause>false</ConsolePause>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="MvvmNano.Core, Version=2.1.0.0, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
We should not reference the debug output of another solution. Instead we should add those two projects to the demo solution as well.